lib/qesapdeployment.pm

```diff
@@ -283,10 +283,11 @@ sub qesap_prepare_env {
     qesap_yaml_replace(openqa_variables => $variables);
     push(@log_files, $paths{qesap_conf_trgt});
 
-    record_info("QESAP conf", "Generating tfvars file");
+    record_info("QESAP conf", "Generating all terraform and Ansible configuration files");
     push(@log_files, "$paths{terraform_dir}/$provider/terraform.tfvars");
     push(@log_files, "$paths{deployment_dir}/ansible/playbooks/vars/hana_media.yaml");
-    push(@log_files, "$paths{deployment_dir}/ansible/playbooks/vars/hana_vars.yaml");
+    my $hana_vars = "$paths{deployment_dir}/ansible/playbooks/vars/hana_vars.yaml";
+    push(@log_files, $hana_vars) if (script_run("test -e $hana_vars") == 0);
```

Comment:
Shouldn't this check go after `qesap_execute(cmd => 'configure')`?

Reply:
Good catch. Yes, you are right. And the proof is the missing of hana_vars.yaml in the list here: https://openqaworker15.qa.suse.cz/tests/56179#step/cluster_configure/111